lib/sensu-handler.rb

```diff
@@ -108,7 +116,9 @@ def api_settings
       return @api_settings if @api_settings
       if ENV['SENSU_API_URL']
         uri = URI(ENV['SENSU_API_URL'])
+        ssl = uri.scheme == 'https' ? {} : nil
```

Comment:
if its https why set it to a hash, just use https and otherwise set it to http, this seems overly complex.

Reply:
See Enterprise API docs https://docs.sensu.io/sensu-enterprise/2.8/api/#reference-documentation Planning to also add keystore/cert support (separate issue/pr).

Reply:
I see, thanks for pointing me to it.